metadata.rb

```diff
@@ -5,10 +5,10 @@
 license           'Apache-2.0'
 description       'Installs and configures postgresql for clients or servers'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '6.1.2'
+version           '7.0.0'
```

Comment:
Do we need to bump the version now?

Reply:
Not really, but it reminds us/people that we're no longer aiming for 6.x series.